### ISSUE TITLE
$climate->table($data) is broken on 8.x

### ISSUE THREAD

**Issue by s22-tech** [thephpleague/climate#203]
Even running your example on https://climate.thephpleague.com/terminal-objects/table/ produces this error:
```
PHP Fatal error:  Uncaught Error: Call to a member function width() on null in /Users/bbb/php/vendor/climate/vendor/league/climate/src/TerminalObject/Basic/Border.php:61
Stack trace:
#0 /Users/bbb/php/vendor/climate/vendor/league/climate/src/TerminalObject/Basic/Table.php(186): League\CLImate\TerminalObject\Basic\Border->result()
#1 /Users/bbb/php/vendor/climate/vendor/league/climate/src/TerminalObject/Basic/Table.php(140): League\CLImate\TerminalObject\Basic\Table->getBorder()
#2 /Users/bbb/php/vendor/climate/vendor/league/climate/src/TerminalObject/Router/BasicRouter.php(28): League\CLImate\TerminalObject\Basic\Table->result()
#3 /Users/bbb/php/vendor/climate/vendor/league/climate/src/TerminalObject/Router/Router.php(99): League\CLImate\TerminalObject\Router\BasicRouter->execute(Object(League\CLImate\TerminalObject\Basic\Table))
#4 /Users/bbb/php/vendor/climate/vendor/league/climate/src/CLImate.php(366): League\CLImate\TerminalObject\Router\Router->execute('table', Array)
#5 /Users/bbb/php/vendor/climate/vendor/league/climate/src/CLImate.php(380): League\CLImate\CLImate->buildTerminalObject('table', Array)
#6 /Users/bbb/php/vendor/climate/vendor/league/climate/src/CLImate.php(417): League\CLImate\CLImate->routeRemainingMethod('table', Array)
#7 /privateuntitled text 169(30): League\CLImate\CLImate->__call('table', Array)
#8 {main}
  thrown in /Users/bbb/php/vendor/climate/vendor/league/climate/src/TerminalObject/Basic/Border.php on line 61
```
macOS 13.5
PHP 8.1, 8.2, 8.3

**Comment by s22-tech** [comment]
The fix to these problems can be found in this fork
[https://github.com/sartor/climate](https://github.com/sartor/climate)

**Comment by duncan3dc** [comment]
This should be fixed by a20112f29623cd66d175e3099a98824f437d21a9
